docs/framework/network-programming/asynchronous-server-socket-example.md

```diff
@@ -144,24 +144,27 @@ using System.Text;
 using System.Threading;  
   
 // State object for reading client data asynchronously  
-public class StateObject {  
-    // Client  socket.  
-    public Socket workSocket = null;  
+public class StateObject {
     // Size of receive buffer.  
-    public const int BufferSize = 1024;  
+    public const int BufferSize = 1024;
```

Comment:
This is unrelated to the PR, but I think it may be recommended to use readonly instead of const here? So, if the class is shipped as part of a package, this integer doesn't get embedded directly in the consumer's IL code. This is useful if the library changed this value later.

@BillWagner, Thoughts?

Reply:
There are valid reasons for both, but this could be readonly. That can be a later fix.